### PR TITLE
[Doc] Improve the dashboard visibility

### PR DESCRIPTION
**Pull request by rkooo567**
Signed-off-by: SangBin Cho <rkooo567@gmail.com>

## Why are these changes needed?

- Remove the tool sub-directory and put all the tools to the top level.
- Move ray dashboard to the top item from monitoring & debugging.
- Add a link to the overview page for all getting started guide.
- Add a observability section to the top-level getting started guide.
- Remove verbose dashboard overview and add a picture instead. Note: I will make an another PR to improve the overview page of the dashboard. 

## Checks

- [ ] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Review by maxpumperla — APPROVED** [review]
Great, that's an improvement for sure!

File: doc/source/ray-overview/index.md [hunk 1]
```diff
@@ -615,6 +615,113 @@ ray submit cluster.yaml example.py --start
...
+```bash
+pip install "ray[default]"
+```
+````
+
+Run the following code.
+
+```{code-block} python
+
+    import ray
```
Comment (jjyao) [inline doc/source/ray-overview/index.md:664]: Is this testable? Do you need to move to doc_code folder?

**Review by jjyao — APPROVED** [review]

File: doc/source/ray-overview/index.md (hunk 1 above)
Comment (rkooo567) [reply doc/source/ray-overview/index.md:664]: It is the same code as the one under state API doc. Let me try moving this.

**Review by alanwguo — COMMENTED** [review]
At least for one of the "Getting started" tutorials, we should not just send them to the dashboard but ask them to look at something.

For example, we can have them click the jobs tab and see their running job and the logs or something.

This can be a follow-up PR though

**Comment by ericl** [comment]
Looks good. Separately, we need to improve our dashboard screenshots to reflect the new IA and show good example workloads (rather than empty job).

**Comment by rkooo567** [comment]
@ericl @alanwguo I am planning to revamp the current dashboard doc to 

1. Have a better overview & getting started guide (that shows features with screenshot)
2. Change all existing screenshots to the new one (hopefully with a couple of gif). 

as a follow up. ETA is EoD today in my time zone.

**Pull request merged**